lib/sparse/src/index/compressed_posting_list.rs

```diff
@@ -32,13 +37,15 @@ pub struct CompressedPostingList<W: Weight> {
 }
 
 /// A non-owning view of [`CompressedPostingList`].
-#[derive(Default, Debug, Clone, PartialEq)]
+#[derive(Debug, Clone)]
 pub struct CompressedPostingListView<'a, W: Weight> {
     id_data: &'a [u8],
     chunks: &'a [CompressedPostingChunk<W>],
     remainders: &'a [GenericPostingElement<W>],
     last_id: Option<PointOffsetType>,
     multiplier: W::QuantizationParams,
+    #[allow(dead_code)] // ToDo: Implement
```

Comment:
```suggestion
    #[allow(dead_code)] // ToDo(io_measurements): Implement
```
Let's add this to make it easier to find all TODOs related to io measurements

Reply:
Actually, this annotation is not needed anymore